ci/redmine_bugzilla.py

```diff
@@ -181,7 +185,28 @@ def main():
                             if bug.status in ['NEW', 'ASSIGNED']:
                                 if external_bug['ext_status'] in ['MODIFIED', 'ON_QA', 'VERIFIED',
                                                          'CLOSED - CURRENTRELEASE']:
-                                    transition_to_post.append(True)
+                                    if 'FailedQA' in bug.cf_verified:
```

Comment:
Do we want this FailedQA branch guarded by L# 185 and L#186? I think maybe we don't, but I'm not totally sure. If we don't it would be moved above L#185 and the old cold L#181-184 would be part of the else block.